From 2f0760aff82c68aad9e6a2107c1fbd798a84a588 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Mon, 27 Jul 2015 16:17:29 +0530
Subject: [PATCH] [EM/Charger]: Set platform max charge current value to 3A

As the platform is capable of supporting USBC PD, it can charge
up to 3A. Currently the platform max charging current is set to 1.8A,
since the DCP can supply up to 1.8A. Based on the platform's charging
capability the max charging current value is changed from 1.8A to 3A.
So the platform cannot be limited the charging current to 1.8A and it
can be charge up to 3A current based on the battery temp zone profile.

Change-Id: Ibd150eb552e42ceb150f7c8321b2ade4e7e09fa0
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-13895
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
---
 drivers/power/bq24192_charger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/bq24192_charger.c b/drivers/power/bq24192_charger.c
index cd521a31bccf..1ea80119d8a6 100644
--- a/drivers/power/bq24192_charger.c
+++ b/drivers/power/bq24192_charger.c
@@ -2557,7 +2557,7 @@ struct bq24192_platform_data tbg24296_drvdata = {
 	.supported_cables = (POWER_SUPPLY_CHARGER_TYPE_USB |
 			POWER_SUPPLY_CHARGER_TYPE_WIRELESS),
 	.sfi_tabl_present = true,
-	.max_cc = 1800,	/* 1800 mA */
+	.max_cc = 3000,	/* 3000 mA */
 	.max_cv = 4350,	/* 4350 mV */
 	.max_temp = 45,	/* 45 DegC */
 	.min_temp = 0,	/* 0 DegC */
-- 
1.9.1

